src/pages/Responses.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div :class="$style.title">自分の回答</div>
+  <div :class="$style.content">
+    <table :class="$style.table">
+      <th
+        v-for="(header, index) in headers"
+        :key="index"
+        :class="$style.header"
+      >
+        {{ header }}
+      </th>
+      <responses-table :response-summaries="responseSummaries.value" />
```

Comment:
```suggestion
      <responses-table :response-summaries="responseSummaries" />
```
valueを書くとはがしてくれます
https://v3.vuejs.org/guide/reactivity-fundamentals.html#ref-unwrapping